cloudinit/config/cc_scripts_vendor.py

```diff
@@ -26,12 +31,18 @@
         prefix: <vendor data prefix>
```

Comment:
We need cloud-init-schema.json defined for cc_scripts_vendor supporting "vendor_data: prefix". Also there is an errant example specifying the "enable: True" is allow but I see no handling of that here.

Reply:
Thanks...I think I skipped over vendor data to understand it better later, and then forgot to come back to it later :man_facepalming: